pkg/cloud/services/network/routetables.go

```diff
@@ -394,3 +379,66 @@ func (s *Service) getRouteTableTagParams(id string, public bool, zone string) in
 		Additional:  additionalTags,
 	}
 }
+
+func (s *Service) getRoutesToPublicSubnet(sn *infrav1.SubnetSpec) ([]*ec2.CreateRouteInput, error) {
+	var routes []*ec2.CreateRouteInput
+
+	if sn.IsEdgeWavelength() {
+		if s.scope.VPC().CarrierGatewayID == nil {
+			return routes, errors.Errorf("failed to create carrier routing table: carrier gateway for %q is nil", s.scope.VPC().ID)
+		}
+		routes = append(routes, s.getCarrierGatewayPublicIPv4Route())
+		// TODO(mtulio): IS IPv6 supported in Carrier Gateway/Wavelength infra?
```

Comment:
From [AWS Docs](https://docs.aws.amazon.com/wavelength/latest/developerguide/wavelength-quotas.html):
> You cannot assign IPv6 addresses to subnets that are in Wavelength Zones.

IPv6 subnets are not supported in Wavelength zones.